sky/backends/backend_utils.py

```diff
@@ -1843,21 +1852,28 @@ def _update_cluster_status(
         return global_user_state.get_cluster_from_name(cluster_name)
 
 
-@timeline.event
-def refresh_cluster_status_handle(
-    cluster_name: str,
-    *,
-    force_refresh: bool = False,
-    acquire_per_cluster_status_lock: bool = True,
-    suppress_error: bool = False
-) -> Tuple[Optional[global_user_state.ClusterStatus],
-           Optional[backends.Backend.ResourceHandle]]:
-    """Refresh the cluster status and return the status and handle.
+def refresh_cluster_record(
+        cluster_name: str,
+        *,
+        force_refresh: bool = False,
+        acquire_per_cluster_status_lock: bool = True
+) -> Optional[Dict[str, Any]]:
+    """Refresh the cluster record.
```

Comment:
```suggestion
    """Refresh the cluster, and return the possibly updated record.
```
Conceptually I feel the difference between the two functions is only what is returned?